src/ui/heatmap/mapChooseFilter.ui.tsx

```diff
@@ -58,25 +57,22 @@ export default function MapChooseFilterUI({ filterBy, setFilterBy, selectfilter,
   return (
     <div className='relative'>
       {/* Filter Button + Name */}
-      <button onClick={() => {setSelectfilter(!selectfilter); setSelectMetaverse(false); setSelectCoord(false);}}>
+      <button onClick={() => { setSelectfilter(!selectfilter); setSelectMetaverse(false); setSelectCoord(false); }}>
         {/* Icon */}
-        <div className={`flex bg-nm-fill dark:bg-nm-dm-fill items-center justify-center rounded-full w-12 h-12 ${selectfilter && "rounded-b-none h-15"}`}>
+        <div className={`flex bg-nm-fill dark:bg-nm-dm-fill items-center justify-center rounded-full w-12 h-12 ${selectfilter && "rounded-b-none h-[60px] pb-3"}`}>
           {filterOptions[filterBy].icon}
         </div>
       </button>
       {/* FilterOptions */}
       {selectfilter && (
         <>
-          <div className='absolute top-[48px] left-[48px] w-3 h-3'>
-            <Image src={'/images/heatmap/curve.svg'} layout='fill' alt='logo' />
-          </div>
           <div className={`flex flex-col space-y-4 absolute bg-nm-fill dark:bg-nm-dm-fill rounded-xl rounded-tl-none p-3 pt-5`}>
             {Object.keys(filterOptions).map((filter) => (
               filter !== filterBy && (
                 <div key={filter}>
                   <button
                     className='flex gap-4 bg-opacity-100 items-center font-medium text-lm-text dark:text-nm-highlight hover:text-nm-dm-remark dark:hover:text-nm-dm-remark min-w-max text-base'
-                    onClick={() => {setFilterBy(filter as MapFilter); setSelectfilter(false)}}
+                    onClick={() => { setFilterBy(filter as MapFilter); setSelectfilter(false) }}
```

Comment:
don't call multiple functions on one line, instead create a function for call multiple functions